### ISSUE TITLE
Archive this feedstock?

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/sphinxcontrib-napoleon-feedstock#5]
It looks like [the upstream repo]( https://github.com/sphinx-contrib/napoleon ) has been archived. Also it was long ago upstreamed into Sphinx ( https://github.com/sphinx-doc/sphinx/commit/a00a096069f44bbb131fe22414fb3053f3ef9fd6 ). Given this, should we archive this feedstock?

cc @conda-forge/sphinxcontrib-napoleon @conda-forge/sphinx

**Comment by AA-Turner** [comment]
> should we archive this feedstock

Yes; makes sense (I wasn't aware one could archive feedstocks).

**Comment by jakirkham** [comment]
Yep we just follow [these steps]( https://github.com/conda-forge/admin-requests#archive-or-unarchive-a-feedstock )
